__tests__/app.test.js

```diff
@@ -214,3 +214,79 @@ describe("POST /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe.only("PATCH /api/reviews/:review_id", () => {
+  test("200: returns the updated review after passing a patch request to update the vote count positively on a valid review_id", () => {
+    return request(app)
+      .patch("/api/reviews/2")
+      .send({
+        inc_votes: 5,
+      })
+      .expect(200)
+      .then(({ body: { comment } }) => {
+        expect(comment.votes).toBe(10);
+        expect(comment).toHaveProperty("review_id");
```

Comment:
this isn't the neatest or most efficient way of doing this, try using the jest matchers .toMatchObject or .objectContaining